mindsdb/integrations/handlers/spotify_handler/spotify_handler.py

```diff
@@ -0,0 +1,50 @@
+from mindsdb.integrations.libs.api_handler import APIHandler, APITable, FuncParser
+
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+
+
+class TwitterHandler(APIHandler):
```

Comment:
why is this here?

Reply:
Work in progress (WIP). I worked on it last night till 3 AM and had to go to bed. Was trying to grasp the concept of handlers. I've already made updates and will push my local changes shortly.